theano/tests/test_tutorial.py

```diff
@@ -139,16 +139,13 @@ def make_node(x, y):
 
         from theano import gof
         class BinaryDoubleOp(gof.Op):
+        
+            __props__ = ("name", "fn")
+            
```

Comment:
I didn't find the corresponding doc to  :         
- class PyCUDADoubleOp(theano.Op):  
- one out of two  class BinaryDoubleOp(gof.Op):